actions/AuthAction.js

```diff
@@ -142,44 +140,34 @@ async function checkForEarlyProductAccess(toggle, sdkKey, isInternal, org) {
   });
 }
 
-function asAuthAction(action) {
+function asAuthAction(action, isAuthorize = true) {
```

Comment:
I added an optional `isAuthorize` parameter to manage authorization checks. If it doesn't make sense to require authorization for certain actions, we can disable it. By default, authorization is enabled for all web actions. I excluded authorization from the following actions:

- `aem-genai-assistant/csv`
- `aem-genai-assistant/target`